tests/test_scripts.py

```diff
@@ -69,7 +69,10 @@ def run_testenv():
 
 
 @pytest.mark.slow
-@pytest.mark.skipif(sys.platform == "win32", reason="causes a freeze in appveyor for some reasons")
+@pytest.mark.skipif(
+    "linux" not in sys.platform,
+    reason="causes a freeze in appveyor for some reasons, and crashes on Catalina",
```

Comment:
ah !

Reply:
It works fine on Big Sur though, there has always been issues with the test_env on Catalina. I guess the `skipif` will remain until we can run tests on a 11.x image